Sources/Mustache/Template+Parser.swift

```diff
@@ -478,7 +508,7 @@ extension MustacheTemplate {
         return state.newLine && self.hasLineFinished(&parser)
     }
 
-    private static let sectionNameCharsWithoutBrackets = Set<Character>("0123456789abcdefghijklmnopqrstuvwxyzABCDEFGHIJKLMNOPQRSTUVWXYZ.-_?")
-    private static let sectionNameChars = Set<Character>("0123456789abcdefghijklmnopqrstuvwxyzABCDEFGHIJKLMNOPQRSTUVWXYZ.-_?()")
+    private static let sectionNameCharsWithoutBrackets = Set<Character>("0123456789abcdefghijklmnopqrstuvwxyzABCDEFGHIJKLMNOPQRSTUVWXYZ.-_?*")
```

Comment:
What's this change for?

Reply:
I'd assume it's for a `*` prefixed section name. Is that the only allowed use case of `*`?

Reply:
Yes it is